### PR TITLE
feat(scout): add support for ScoutAPM extension

### PR DESCRIPTION
**Pull request by Jean-Serge**
Hello,

This PR enables support to install the [Scout APM](https://scoutapm.com/) extension.

### CODE REVIEW

**Comment by EtienneM** [comment]
Hi,

Thanks a lot for this PR. We will have a look at it in the coming weeks.

File: README.md [hunk 1]
```diff
@@ -381,6 +381,26 @@ Example:
...
+
+Enable support for [Scout APM](https://scoutapm.com/).
+
+To enable Scout support, set the `extra.paas.scout` property to `true`:
+
+```
+    "scout": true
+```
+
+To install a specific version of the extension, use the `SCOUT_APM_VERSION` environment variable (`1.6.0` by default).
```
Comment (EtienneM) [inline README.md:396]:
```suggestion
To install a specific version of the extension, use the `SCOUT_APM_VERSION` environment variable (`1.7.0` by default).
```

File: README.md [hunk 2]
```diff
@@ -381,6 +381,26 @@ Example:
...
+
+Example:
+
+```
+SCOUT_APM_VERSION=1.6.0
```
Comment (EtienneM) [inline README.md:401]:
```suggestion
SCOUT_APM_VERSION=1.7.0
```

File: bin/compile [hunk 3]
```diff
@@ -238,6 +249,7 @@ NGINX_INCLUDES="${NGINX_INCLUDES:-}"
 ACCESS_LOG_FORMAT_COMPOSER_JSON="${ACCESS_LOG_FORMAT_COMPOSER_JSON:-}"
 DATADOG_TRACER_VERSION="${DATADOG_TRACER_VERSION:-latest}"
 DATADOG_APPSEC_VERSION="${DATADOG_APPSEC_VERSION:-0}"
+SCOUT_APM_VERSION=${SCOUT_APM_VERSION:-1.6.0}
```
Comment (EtienneM) [inline bin/compile:252]:
```suggestion
SCOUT_APM_VERSION=${SCOUT_APM_VERSION:-1.7.0}
```

File: lib/scout [hunk 4]
```diff
@@ -0,0 +1,23 @@
+function install_scout() {
+    status "ScoutAPM usage detected, installing PHP extension"
+
+    local ext_dir=/app/vendor/php/lib/php/extensions/no-debug-non-zts-$(php_api_version)
+    local version="${1}"
+    local cwd=$(pwd)
+    local temp_dir=$(mktmpdir "scout")
+
+
```
Comment (EtienneM) [inline lib/scout:9]: nitpick:
```suggestion
```

File: lib/scout [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+    cd "${temp_dir}"
+
+    curl -L "https://pecl.php.net/get/scoutapm-${version}.tgz" | tar xzv
```
Comment (EtienneM) [inline lib/scout:12]: I prefer we use long option in scripts:
```suggestion
    curl --location "https://pecl.php.net/get/scoutapm-${version}.tgz" | tar xzv
```

**Review by EtienneM — COMMENTED** [review]
Version 1.7.0 just got released, let's make it the default. Otherwise looks good to me, thanks :)

**Comment by Jean-Serge** [comment]
Oops, should have made batch resolution for your suggestion. Do you want me to squash those @EtienneM  ?

**Comment by EtienneM** [comment]
That would be lovely, yes :)

**Comment by EtienneM** [comment]
@Jean-Serge support for ScoutAPM extension is released in prod.

**Pull request merged**